File: android/src/main/java/com/yugioh/android/DeckCardActivity.java
package com.yugioh.android;

import android.app.Fragment;
import com.yugioh.android.base.BaseActivity;
import com.yugioh.android.fragments.DeckCardFragment;

public class DeckCardActivity extends BaseActivity {
    @Override
    public int getIcon() {
        return R.drawable.icon;
    }

    @Override
    public Fragment replaceFragment() {
        DeckCardFragment dcf = new DeckCardFragment();
        dcf.setArguments(getIntent().getExtras());
        return dcf;
    }

    @Override
    public int customTheme() {
        return 0;
    }
}

File: android/src/main/kotlin/com/yugioh/android/DeckCardActivity.kt
package com.yugioh.android

import android.app.Fragment
import com.rarnu.base.app.BaseActivity
import com.yugioh.android.fragments.DeckCardFragment

class DeckCardActivity : BaseActivity() {

    override fun getActionBarCanBack(): Boolean = true

    override fun getIcon(): Int = R.drawable.icon

    override fun replaceFragment(): Fragment {
        val dcf = DeckCardFragment()
        dcf.arguments = intent.extras
        return dcf
    }

    override fun customTheme(): Int = 0

}
